### PR TITLE
Detect dev app server correctly even if it's not localhost.

### PR DESCRIPTION
**Pull request by gimite**

### CODE REVIEW

File: app/utils.py [hunk 1]
```diff
@@ -474,6 +474,9 @@ def strip_url_scheme(url):
     _, netloc, path, query, segment = urlparse.urlsplit(url)
     return urlparse.urlunsplit(('', netloc, path, query, segment))
```
Comment (skywhale) [inline app/utils.py]: 2 blank lines between top-level functions

**Comment by skywhale** [comment]
LGTM

File: app/utils.py (hunk 1 above)
Comment (gimite) [inline app/utils.py]: Done.

**Comment by gimite** [comment]
Thanks for the review. Merged.

**Pull request merged**